os/tools/check_package_size.py

```diff
@@ -32,6 +32,7 @@
 output_folder = build_folder + '/output/bin'
```

Comment:
Please change "usaed" --> "used" in the ```The usaed ratio means (Binary Size)/(Partition Size)``` part of commit message.

Reply:
Thank you, Fixed!